src/SalmonDE/TopVoter/Tasks/UpdateVotesTask.php

```diff
@@ -23,6 +26,18 @@ public function unsetKey(): void{
 	}
 
 	public function onRun(int $currentTick): void{
+	    if($this->usePocketVote) {
```

Comment:
The formatting in this part seems to have gotten out of hand. Please fix it.
Especially the tabs spaces mix. The prefered method of indentation is tabs.

Also I'd suggest creating a new task class for PocketVote support. Otherwise there would be unused fields like key and amount.